api_factory/generator/templates/$namespace/$name_$version/$service/transports/base.py.j2

```diff
@@ -42,6 +45,13 @@ class {{ service.name }}Transport(metaclass=abc.ABCMeta):
 
         # Save the credentials.
         self._credentials = credentials
+    {%- if service.has_lro %}
+
+    @abc.abstractproperty
```

Comment:
In its current state, this will actually make the HTTP transport not work (it will not initialize) for any API with an LRO in it, even if that API is not using LRO.

This exposes an issue with our testing (which we already knew about) since our tests do not catch such a thing.
It also means we need a provisional LRO client for proto-over-HTTP. 

Since this is pre-alpha, I think it would be sufficient to just return the gRPC-based OperationsClient from the HTTP transport for now, and we can revisit later. (There are other bugs that still require `grpcio` to be installed at the moment, so that at least will not make it any worse.)